src/app/components/Toolbar/Features/Contacts/DeleteContact.tsx

```diff
@@ -0,0 +1,48 @@
+import { useContext } from 'react'
+import { useTranslation } from 'react-i18next'
+import { Box } from 'grommet/es6/components/Box'
+import { Button } from 'grommet/es6/components/Button'
+import { ResponsiveContext } from 'grommet/es6/contexts/ResponsiveContext'
+import { Text } from 'grommet/es6/components/Text'
+import { ResponsiveLayer } from '../../../ResponsiveLayer'
+
+interface DeleteContactProps {
+  deleteHandler: () => void
+  setLayerVisibility: (isVisible: boolean) => void
+}
+
+export const DeleteContact = ({ deleteHandler, setLayerVisibility }: DeleteContactProps) => {
+  const { t } = useTranslation()
+  const isMobile = useContext(ResponsiveContext) === 'small'
+
+  return (
+    <ResponsiveLayer
+      onClickOutside={() => setLayerVisibility(false)}
+      onEsc={() => setLayerVisibility(false)}
+      animation="none"
+      background="background-front"
+      modal
+      margin={isMobile ? 'none' : 'xlarge'}
+    >
+      <Box margin="medium">
+        <Box flex="grow" justify="center">
+          <Text weight="bold" size="medium" alignSelf="center" margin={{ bottom: 'large' }}>
+            {t('toolbar.contacts.delete', 'Delete Contact')}
```

Comment:
I would have a separate translation string for title and button label. They are semantically very different and "Confirm contact deletion?" would be a valid alternative title but not label.

Same pattern in profile deletion: https://github.com/oasisprotocol/oasis-wallet-web/blob/cd2b836864fa1507994fe946e34e827e01002261/src/locales/en/translation.json#L365-L370